frameworks-versions.properties

```diff
@@ -84,28 +88,35 @@ crossEngine.spark33flink115.sparkMajorVersion=3.3
 crossEngine.spark33flink115.scalaMajorVersion=2.12
 crossEngine.spark33flink115.flinkMajorVersion=1.15
 
+crossEngine.spark33flink116.sparkMajorVersion=3.3
```

Comment:
should `spark33flink116` be added to `crossEngineSetups` above as well? or was this intentionally omitted?

Reply:
oops, added